sphinx/domains/std.py

```diff
@@ -302,13 +303,18 @@ class Glossary(SphinxDirective):
 
     has_content = True
     required_arguments = 0
-    optional_arguments = 0
+    optional_arguments = 1
     final_argument_whitespace = False
```

Comment:
Enabling this allows users to use phrases to glossary key.

```
.. glossary:: document writing
```

Reply:
@tk0miya What do you suggest I do here?

Should I allow this behavior?

Or, check for spaces, and if there are spaces, throw an error?